sklearn/tests/test_common.py

```diff
@@ -45,29 +45,40 @@ def test_all_estimator_no_base_class():
         assert_false(name.lower().startswith('base'), msg=msg)
 
 
-def test_all_estimators():
-    estimators = all_estimators(include_meta_estimators=True)
-
-    # Meta sanity-check to make sure that the estimator introspection runs
-    # properly
-    assert_greater(len(estimators), 0)
-
-
 @pytest.mark.parametrize(
         'name, Estimator',
-        all_estimators(include_meta_estimators=True)
+        all_estimators()
 )
 def test_parameters_default_constructible(name, Estimator):
     # Test that estimators are default-constructible
     check_parameters_default_constructible(name, Estimator)
 
 
-def _tested_non_meta_estimators():
+def _tested_estimators():
     for name, Estimator in all_estimators():
         if issubclass(Estimator, BiclusterMixin):
             continue
         if name.startswith("_"):
             continue
+        # FIXME _skip_test should be used here (if we could)
+
+        required_parameters = getattr(Estimator, "_required_parameters", [])
+        if len(required_parameters):
+            continue
+            """
```

Comment:
aarrgh just saw this. this is the exciting part and it's disabled? hm... damn I'll see if we can actually make this work, but maybe that needs to wait for another PR.